cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricStore.java

```diff
@@ -84,6 +85,19 @@ public Cube get() {
       // service
       Constants.Metrics.Tag.SERVICE_RUNNABLE, PROGRAM_LEVEL2
     );
+
+    // reverse mapping
+    // todo : remove hard-coded values and use TypeId (need to move TypeId out of app-fabric ?)
+    this.reverseMapping = ImmutableMap.<String, Map<String, String>>of(
+      //flow
+      "f", ImmutableMap.<String, String>of(PROGRAM_LEVEL2, Constants.Metrics.Tag.FLOWLET,
```

Comment:
yes , but that adda an dependency on app-fabric, wasn't sure if that's the right thing to do.